deepchem/utils/pytorch_utils.py

```diff
@@ -46,11 +46,13 @@ def unsorted_segment_sum(data: torch.Tensor, segment_ids: torch.Tensor,
         [5., 6., 7., 8.]])
 
     """
-    # length of segment_ids.shape should be 1
-    assert len(segment_ids.shape) == 1
 
-    # Shape of segment_ids should be equal to first dimension of data
-    assert segment_ids.shape[-1] == data.shape[0]
+    if len(segment_ids.shape) != 1:
+        raise AssertionError("segment_ids have be a 1-D tensor")
+
+    if data.shape[0] != segment_ids.shape[0]:
```

Comment:
Can you add to the docs examples providing correct `data.shape` and `segment_ids.shape` values?